test/Test/Management.hs

```diff
@@ -0,0 +1,111 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.Management
+  ( test_baseDAO
+  ) where
+
+import Universum
+
+import Lorentz
+import Lorentz.Test
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase)
+
+import Lorentz.Contracts.BaseDAO (Storage(..), mkStorage)
+import Michelson.Runtime (ExecutorError)
+import Test.Common
+import Util.Named
+
+testOwner, wallet1, wallet2 :: Address
+testOwner = genesisAddress1
+wallet1 = genesisAddress2
+wallet2 = genesisAddress3
+
+-- | We test non-token entrypoints of the BaseDAO contract here
+test_baseDAO :: [TestTree]
+test_baseDAO =
+  [ testGroup "Ownership transfer"
+    [ testCase "Transfer ownership entrypoint authenticates sender" $
+        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
```

Comment:
Integrational interface is being replaced with nettest, and in new code we try to stick to nettest. See how tests in `Test.BaseDAO.FA2` are written. It should be relatively easy to transform these tests to use nettests.

After we do https://gitlab.com/morley-framework/morley/-/issues/365, we'll be able to run all these tests in a real network.

Reply:
Yes, I thought about it, but figured Integrational tests could work for now. Have changed it.